features/product/managing_products/deleting_product.feature

```diff
@@ -2,7 +2,7 @@
 Feature: Deleting a product
     In order to remove test, obsolete or incorrect products
     As an Administrator
-    I want to be able to delete products from the product catalog
+    I want to be able to delete products from the product catalogue
```

Comment:
I think we actually use more of American english, and default locale and currency is USD. I do not have preference over any, but let's keep ``catalog``, as this is how we referenced this everywhere and I think we should stick to that.